src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/AbstractTableDataSourceFindUsagesContext.cs

```diff
@@ -397,15 +387,27 @@ private async Task<(ExcerptResult, SourceText)> ExcerptAsync(SourceText sourceTe
                 //   { "Column1", "Value1, Value2" },
                 //   { "Column2", "Value3, Value4" }
                 // }
+                if (customColumnsWithMultipleValues != null && customColumnsWithMultipleValues.Any())
+                {
+                    foreach (var column in customColumnsWithMultipleValues)
```

Comment:
Original code had the check `_customColumnTitleToStatesMap.ContainsKey(kvp.Key)` to ensure that subsequent `GetCustomColumn` call always succeeds. You should probably retain this check:

```csharp
if (_customColumnTitleToStatesMap.ContainsKey(column.Key))
{
   builder.Add(...
}
```